lib/assets/data_transform_cell/main.js

```diff
@@ -543,26 +543,26 @@ export async function init(ctx, payload) {
                           />
                         </div>
                         <div class="row" v-if="operation.operation_type === 'summarise'">
+                          <BaseSelect
+                            name="query"
+                            operation_type="summarise"
+                            label="Summarise using"
+                            v-model="operation.query"
+                            :options="summariseUsingOptions"
+                            :index="index"
+                            :disabled="noDataFrame"
+                          />
                           <BaseMultiTagSelect
                             @change.self="addInnerValue($event)"
                             operation_type="summarise"
-                            label="Summarise by"
+                            label="By columns"
```

Comment:
I am not sure if this is correct. We may need "Of columns". I will check with a native english speaker.

Reply:
How about just "Columns"?

Reply:
I am thinking either "Columns" or "Of columns". I will let you know. :)

Reply:
```suggestion
                            label="Of columns"
```